### PR TITLE
feat: add additional common configs

### PR DESCRIPTION
**Pull request by LaszloGombos**
## Changes

This PR makes it easier and consistent to package common configurations for distribution. 

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

### CODE REVIEW

File: dracut.conf.d/50-generic.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# generic initrd works on other platforms than the one dracut was called
+
```
Comment (bdrung) [inline dracut.conf.d/50-generic.conf:2]: I would remove the empty line between the comment and the config value.

File: dracut.conf.d/50-hostonly.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# as small as possible initrd for improved performance
+
```
Comment (bdrung) [inline dracut.conf.d/50-hostonly.conf:2]: same here.

File: dracut.conf.d/50-generic.conf (hunk 1 above)
Comment (LaszloGombos) [reply dracut.conf.d/50-generic.conf]: ok, thanks.

File: dracut.conf.d/50-hostonly.conf (hunk 2 above)
Comment (LaszloGombos) [reply dracut.conf.d/50-hostonly.conf]: ok, thanks.

**Review by bdrung — APPROVED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**